decidim-core/app/assets/stylesheets/decidim/utils/_flash.scss

```diff
@@ -0,0 +1,3 @@
+.flash{
```

Comment:
I do not see the need of adding a new file just for this. Is this not achievable with CSS classes?
Take a look at the mixin [here](https://github.com/decidim/decidim/blob/master/decidim-core/app/assets/stylesheets/decidim/modules/_margins.scss). You dispose a lot of helper classes, in this case, appending `mb-none` should work.

Reply:
@Crashillo So you mean that I should add the `mb-none` class to the flash template instead? I can do that, no problem. I guess I should then remove the "flash" CSS class from the template a well? I assumed it was there for styling the flash, but it seems like it's not so what other purpose could it have?

Reply:
If I'm not wrong, that `flash` is actually the `callout` component from foundation, isn't it? If so, yeah, remove both, the whole file and the reference in `_decidim`. Restore the template class name, appending the class I mentioned.

The helper purpose is to separate the element/component/partial/whatever from its paddings/margins to avoid this kind of problems when we reutilisate them all along the app, adding/removing the helper classes according to our needs.

Reply:
Note that I haven't changed the flash template at all, I'm just styling a "flash" class that was already defined... The flash template markup includes both the foundation "callout" CSS class and the custom "flash" CSS class, so it seemed perfect for this use case where I want to customize the appearance of all flashes, but not all callouts.

Anyways, I can't really remove the "flash" class since some tests rely on it. So I think I know now the exact diff you're looking for. Hold on.

Reply:
Actually, flashes are rendered from a custom helper defined in an external gem, so it's a bit problematic to do what you're asking: https://github.com/sgruhier/foundation_rails_helper/blob/8a4b53d9b348bbf49c3608aea811fe17b9282e4c/lib/foundation_rails_helper/flash_helper.rb#L1. Let me know how you want me to proceed.

Reply:
😖 that escapes from my understanding. I was taking a look at the flash, and the way to render it (`display_flash_messages` right?).

1. Is it possible call the function `display_flash_messages(closable: true, key_matching: {})` with a parameter in our views? (and pass it our `mb-none` class) cc/ @decidim/lot-core this would be the better approach.
2. If not, it comes to my mind in the callout component we already have, add `.callout.flash { margin-botton: 0; }`, always on flash comes together callout class... they're different things but if they're always displayed together...

What do you think @deivid-rodriguez & @decidim/lot-core, is it possible or too much?